data/transport missions.txt

```diff
@@ -1975,3 +1975,40 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "The Broker"
+	description "Transport the broker's enforcers to <destination> by <day>."
+	blocked "You'll need more room to house the hired thugs. Return when you have <capacity>."
+	deadline
+	cargo "Living necessities" 5
+	passengers 15
+	minor
+	to offer
+		not "year" == 3013
+	source "New Boston"
+	destination
+		government "Syndicate"
+	on offer
+		conversation
+			'As you're walking through the spaceport searching for a bar or pub, someone catches your eye. A nearly bald, middle-aged man wanders through the crowd in an expensively tailored suit, searching the faces that pass by him. You stop for a moment and look closer; he seems familiar in some way. Eventually, his gaze settles on you, making eye contact. He blinks a few times in surprise, then begins moving through the crowd towards you.'
+			choice
+				'	(Turn the other way and lose the man in the crowd.)'
+					decline
+				'	(Wait for him to reach you.)'
+				'	(Walk forwards to meet him.)'
```

Comment:
"Forward", with no s, is preferred in American English, which is what the game is written in.
https://books.google.com/ngrams/graph?content=toward%2Ctowards&case_insensitive=on&year_start=1800&year_end=2008&corpus=17&smoothing=3&share=&direct_url=t4%3B%2Ctoward%3B%2Cc0%3B%2Cs0%3B%3Btoward%3B%2Cc0%3B%3BToward%3B%2Cc0%3B.t4%3B%2Ctowards%3B%2Cc0%3B%2Cs0%3B%3Btowards%3B%2Cc0%3B%3BTowards%3B%2Cc0